### PR TITLE
[fix #4] [mozmeao/infra#1104] Clean up oregon-b dev yaml.

### LINKED ISSUE #4: clean up yaml for oregon-b dev
related to mozmeao/infra#1104

### PR DESCRIPTION
**Pull request by jpetto**
- rename svc.yaml to web-svc.yaml

### CODE REVIEW

File: oregon-b/basket-dev/donateworker-deploy.yaml [hunk 1]
```diff
@@ -48,376 +37,231 @@ spec:
...
-              key: workflow-release-summary
-              name: basket-dev-v125-env
+          value: "True"
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-dev/donateworker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-dev/donateworker-deploy.yaml [hunk 2]
```diff
@@ -48,376 +37,231 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-dev/donateworker-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: oregon-b/basket-dev/ns.yaml [hunk 3]
```diff
@@ -1,13 +1,7 @@
...
 kind: Namespace
 metadata:
-  creationTimestamp: "2018-01-24T22:04:17Z"
-  labels:
-    heritage: deis
   name: basket-dev
-  resourceVersion: "1120334"
-  selfLink: /api/v1/namespaces/basket-dev
-  uid: 857dddb5-0152-11e8-8c8b-0656fabf84c0
 spec:
```
Comment (jgmize) [inline oregon-b/basket-dev/ns.yaml]: spec and status sections can be removed too

File: oregon-b/basket-dev/web-deploy.yaml [hunk 4]
```diff
@@ -48,376 +37,231 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-dev-v125-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-dev-v125-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-dev/web-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-dev/web-deploy.yaml [hunk 5]
```diff
@@ -48,376 +37,231 @@ spec:
...
-            secretKeyRef:
-              key: port
-              name: basket-dev-v125-env
+              name: basket-dev-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-dev-v125-env
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-dev/web-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-dev/web-deploy.yaml [hunk 6]
```diff
@@ -48,376 +37,231 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-dev-v125-env
+          value: 0
```
Comment (jgmize) [inline oregon-b/basket-dev/web-deploy.yaml]:
```suggestion
          value: "0"
```

File: oregon-b/basket-dev/web-deploy.yaml [hunk 7]
```diff
@@ -48,376 +37,231 @@ spec:
...
-              key: workflow-release-summary
-              name: basket-dev-v125-env
+          value: "True"
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-dev/web-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-dev/web-deploy.yaml [hunk 8]
```diff
@@ -48,376 +37,231 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-dev/web-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: oregon-b/basket-dev/worker-deploy.yaml [hunk 9]
```diff
@@ -48,376 +37,231 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-dev-v125-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-dev-v125-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-dev/worker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-dev/worker-deploy.yaml [hunk 10]
```diff
@@ -48,376 +37,231 @@ spec:
...
-            secretKeyRef:
-              key: port
-              name: basket-dev-v125-env
+              name: basket-dev-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-dev-v125-env
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-dev/worker-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-dev/worker-deploy.yaml [hunk 11]
```diff
@@ -48,376 +37,231 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-dev-v125-env
+          value: 0
```
Comment (jgmize) [inline oregon-b/basket-dev/worker-deploy.yaml]:
```suggestion
          value: "0"
```

File: oregon-b/basket-dev/worker-deploy.yaml [hunk 12]
```diff
@@ -48,376 +37,231 @@ spec:
...
-              key: workflow-release-summary
-              name: basket-dev-v125-env
+          value: "True"
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-dev/worker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-dev/worker-deploy.yaml [hunk 13]
```diff
@@ -48,376 +37,231 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-dev/worker-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

**Review by jgmize — CHANGES_REQUESTED** [review]
see comments on https://github.com/mozmeao/basket-config/pull/20 -- we need to make the same changes here and in the other PRs

**Comment by jpetto** [comment]
@jgmize updated!

**Review by jgmize — APPROVED** [review]

**Comment by jgmize** [comment]
thanks again @jpetto!

**Pull request merged**